static/src/js/components/EarthdataDownload/EarthdataDownload.js

```diff
@@ -0,0 +1,204 @@
+  const windowsDownloadLink = '//github.com/nasa/earthdata-download/releases/latest/download/Earthdata-Download-x64.exe'
+  const macDownloadLink = '//github.com/nasa/earthdata-download/releases/latest/download/Earthdata-Download-x64.dmg'
+  const macSiliconDownloadLink = '//github.com/nasa/earthdata-download/releases/latest/download/Earthdata-Download-arm64.dmg'
+  const linuxDownloadLink = '//github.com/nasa/earthdata-download/releases/latest/download/Earthdata.Download-amd64.deb'
+  const {
+    macOsEDDExecutableSize,
+    windowsEDDExecutableSize,
+    linuxEDDExecutableSize
+  } = getApplicationConfig()
+
+  let isMacOS = false
+  let isLinux = false
+  let isWindows = false
+
+  switch (operatingSystem) {
+    case 'macOS': {
+      // Apple standard is not to capitalize macOS
+      isMacOS = true
+      downloadLink = macDownloadLink
+      executableSize = macOsEDDExecutableSize
+      break
+    }
+    case 'Windows': {
+      isWindows = true
+      downloadLink = windowsDownloadLink
+      executableSize = windowsEDDExecutableSize
+      break
+    }
+    case 'Linux': {
+      isLinux = true
+      downloadLink = linuxDownloadLink
+      executableSize = linuxEDDExecutableSize
+      break
+    }
+    default: {
+      operatingSystem = 'macOS'
+      isMacOS = true
+      downloadLink = macDownloadLink
+      executableSize = macOsEDDExecutableSize
+      break
+    }
+  }
+  const downloaderSize = `${executableSize}mb`
+  const osLinkFileExt = `.${downloadLink.split('.').pop()}`
+
+  return (
+    <div className="earthdata-download container">
+      <section className="mb-5 mt-4">
+        <h2 className="font-weight-bolder text-center">Earthdata Download</h2>
+        <span className="h3 d-block font-weight-light text-black-50 text-center">
+          Download your Earth science data from Earthdata Search with only one click
+        </span>
+      </section>
+      <section className="d-flex flex-column align-items-center mt-4">
+        <Button
+          dataTestId="eddDownloadButton"
+          className="earthdata-download__install-button"
+          type="button"
+          icon={FaDownload}
+          bootstrapVariant="success"
+          bootstrapSize="lg"
+          href={downloadLink}
+          download
+        >
+          Download for
+          {' '}
+          {operatingSystem}
+        </Button>
+        <div className="mt-2 text-black-50">
+          <span className="earthdata-download__download-size">
+            {downloaderSize}
+          </span>
+          {' '}
+          <span className="earthdata-download__os-link-file-ext">
+            {osLinkFileExt}
+          </span>
+          {
+            operatingSystem === 'macOS' && (
+              <span>{' (for Intel-based Macs)'}</span>
+            )
+          }
+        </div>
+      </section>
+      <section className="row my-5">
+        {
+        isMacOS
+          ? (
+            <div className="col-md align-items-center d-flex flex-column">
```

Comment:
For conditionals with elements, I would prefer to see any kind of logic outside the display block.

```
let isMacElement = isMac? <div...> : <div> other</div>

return (
<div id="mainElement">
{ isMacElement }
</div>
)
```

Reply:
Switched to this format